elintegrostartapp/grails-app/services/elintegroWebsite/ContactUsMessageService.groovy

```diff
@@ -0,0 +1,26 @@
+package elintegroWebsite
+
+class ContactUsMessageService {
+    def emailService
+    def mailService
+    def sendingMessages() {
+        // println("Job Run Inside Service")
+        def contactDetails = ContactUs.findAllWhere(sendNo:0)
+       // def contactDetails = ContactUs.list(sendNo:1, resend:true)(where vitrako condition rakhda kam garena)
+        println(contactDetails.email)
+        println("This is message from contact us")
+//        emailService.sendMail(email, emailParams, emailBody) {
+//        }
+//        mailService.sendMail{
+//            from "elintegro.himalaya@gmail.com"
+//            to "pangenirabindra5@gmail.com"
```

Comment:
Good, but we do not want to keep your email address hard-coded in the source code. So please create a configuration variable (I believe in application.groovy)

elintegro.hr.emails = emaiol1@elintegro.com, email2.gmail.com,...

this variable can be defined per environment, so in production we will have different list of emails thwn in qa or development environments, but we want to have one default, generic. Check with Prakash or Kumar where is the right place to put this and how to access them from your code.
Later on we will override this lists from the database.